src/t_string.c

```diff
@@ -305,33 +337,34 @@ int parseExtendedStringArgumentsOrReply(client *c, int *flags, int *unit, robj *
 }
 
 /* SET key value [NX] [XX] [KEEPTTL] [GET] [EX <seconds>] [PX <milliseconds>]
- *     [EXAT <seconds-timestamp>][PXAT <milliseconds-timestamp>] */
+ *     [EXAT <seconds-timestamp>][PXAT <milliseconds-timestamp>] [IFEQ comparison-value] */
```

Comment:
We should correct this comment to the correct syntax:

    SET key value [NX | XX | IFEQ comparison-value] [GET]
        [EX seconds | PX milliseconds |
         EXAT unix-time-seconds | PXAT unix-time-milliseconds |
         KEEPTTL]